src/IW/Core/PGArray'.hs

```diff
@@ -0,0 +1,22 @@
+{-# LANGUAGE DeriveAnyClass #-}
+
+module IW.Core.PGArray'
+    ( PGArray' (..)
+    ) where
+
+import Database.PostgreSQL.Simple.Types (PGArray (..))
+import Database.PostgreSQL.Simple.FromField (FieldParser, pgArrayFieldParser)
+import Database.PostgreSQL.Simple.ToField (Action (..))
+
+
+newtype PGArray' a = PGArray' { fromPGArray' :: [a] }
+    deriving stock (Generic, Show, Eq)
+    deriving anyclass (ToJSON)
+
```

Comment:
I wonder if you can use `DerivingVia` to reduce some boilerplate. And write something like:

```haskell
    deriving (ToField, FromField) via PGArray a
```

instead of these two manual instances.

Reply:
I will try this.